### PR TITLE
Add a veyron-speedy ARM port of dpup bullseye

### PR DESCRIPTION
**Pull request by dimkr**
This is a straight port of Puppy Linux to the Asus C201: a "clean" version of #1948 that doesn't introduce PUPMODE 6 (with https://github.com/dimkr/frugalify).

It behaves like Puppy for Raspberry Pi: it's a "full installation", but the images produced by woof-CE use the required Chrome OS partition layout, and the kernel built by #2226.

~(I'll un-draft this PR after testing.)~ done

### CODE REVIEW

File: woof-distro/arm/debian/bullseye-veyron-speedy/_00build_2.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# but override these settings:
+
+## Download and include custom SFS (XDRV_INC= overrides this)
+#ADRV_SFS_URL=
+#YDRV_SFS_URL=
+#FDRV_SFS_URL=
+
+ADRV_INC=
+BUILD_CROS_IMAGE=yes
+BOOT_BOARD='veyron-speedy'
```
Comment (dimkr) [inline woof-distro/arm/debian/bullseye-veyron-speedy/_00build_2.conf:13]: Like woof-distro/x86/debian/bullseye, woof-distro/arm/debian/bullseye-veyron-speedy "inherits" from woof-distro/x86_64/debian/bullseye64. All files except DISTRO_SPECS and _00build_2.conf are symlinks. #2033 and #2234 allow this elegant reuse, and the dpup rolling release in my woof-CE fork "inherits" from each of the three architectures (x86_64, x86 and veyron-speedy) while also setting the theme in _00build_2.conf.

**Review by 01micko — COMMENTED** [review]
It looks sane to me, but I'd like some other eyes to go over it before I merge.

**Comment by peabee** [comment]
Sorry - not qualified to comment..... no idea what "veyron-speedy" is or what is its significance.... how is this "better" than the existing woof-distro/arm/raspbian/buster other than being bullseye instead of buster?

**Comment by 01micko** [comment]
@peabee _every single arm board_ is different requiring different kernel configuration and boot loaders (among other things like boot medium). All I want to confirm is that nothing here breaks any existing routines. As I said, "it looks sane to me".

**Comment by dimkr** [comment]
> Sorry - not qualified to comment..... no idea what "veyron-speedy" is or what is its significance.... how is this "better" than the existing woof-distro/arm/raspbian/buster other than being bullseye instead of buster?

The Asus C201 Chromebook's codename in Chrome OS is veyron-speedy.

Noting to do with Raspberry Pi, Raspbian or Debian Buster.

**Pull request merged**